migrator.go

```diff
@@ -0,0 +1,84 @@
+package influxdb
+
+import "context"
+
+// MigrationManager is the manager of migrators.
+type MigrationManager struct {
+	Migrators []Migrator
+}
+
+// Migrate will perform the migration.
+func (mg MigrationManager) Migrate(ctx context.Context, Up bool, errCh chan<- error) {
```

Comment:
If we are going to use a channel its best to have the `Migrate` function return a channel. `Migrate(ctx context.Context, up bool) (chan <- error) `. But if this isn't being executed in a separate go routine there is no reason to use channels.